packages/backend/src/apps/tiles/actions/update-row/index.ts

```diff
@@ -103,8 +136,20 @@ const action: IRawAction = {
       columnIds,
     })
 
+    /**
+     * Row ID does not correspond to a row, we short circuit the action but do not fail it.
```

Comment:
nit: what's the use case for this? It feels like we're inconsistent between columns and rows. We error out on invalid column IDs, but OK for invalid row IDs.